package.json

```diff
@@ -2,8 +2,10 @@
   "name": "restaurant-page",
   "version": "1.0.0",
   "description": "A simple restaurant page",
-  "main": "index.js",
+  "private": true,
   "scripts": {
+    "build": "webpack",
+    "watch": "webpack --watch",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
**[REQUIRED]**
We need to have a dev or develop script for the development stage.
You can do something like: `"dev": "webpack-dev-server --mode development"`
This way you are telling webpack to run in the develpment mode.

For the build script you can also specify the mode: `"build": "webpack --mode production"`

**[OPTIONAL]**
 You can build a script to run the application in a server in the develop mode with live updates after you change your code: `"start": "webpack-dev-server --watch /src --mode development"`
This will run your app in the development mode on a server and watch for changes(you will have live updates after changing the code).

Reply:
Hi Torres, can you send me a link from the assignment that make the use of webpack-dev-serve a requirement?